From 18bd7e66b720ea4c734cfe6fe09c3bb81ce0c32b Mon Sep 17 00:00:00 2001
From: CanYang He <hcy@rock-chips.com>
Date: Fri, 11 May 2018 17:32:46 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399 lpddr4 dts add center-supply

rk3399-evb-rev3-android-lp4.dts add center-supply, otherwise customer
follow this dts file may leave out it. and cause
rockchip_dmcfreq_probe() fail, then lpddr4 lose scale frequency function
and cannot find /sys/class/devfreq/dmc

Change-Id: I05f36a80fd5f95e29f1083f956a799149e88b1f3
Signed-off-by: CanYang He <hcy@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
index 3bbe52392792..8dd0a0bf36ee 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
@@ -111,6 +111,7 @@
 
 &dmc {
 	status = "okay";
+	center-supply = <&vdd_center>;
 	system-status-freq = <
 		/*system status         freq(KHz)*/
 		SYS_STATUS_NORMAL       800000
-- 
2.35.3

